### PR TITLE
MAUI- 886571 Prepared UG for Smart axis label alignment support in .NET MAUI SfCartesian Chart

### PR DESCRIPTION
**Pull request by Sowndharya1911**
# Description

- Prepared UG for Smart axis label alignment support in .NET MAUI SfCartesian Chart

### CODE REVIEW

File: MAUI/Cartesian-Charts/Axis/Axislabels.md [hunk 1]
```diff
@@ -166,4 +166,79 @@ chart.XAxes.Add(primaryAxis);
 
 {% endhighlight %}
 
+{% endtabs %}
+
+## Smart Axis Labels
+
+Axis labels may overlap with each other based on chart dimensions and label size. The [LabelsIntersectAction]() property of axis is used to avoid overlapping of axis labels. The default value of the LabelsIntersectAction is `None`; other available values are `MultipleRows`, `Hide`, and `Wrap`.
```
Comment (VimalaThirumalaiKumarSF3739) [inline MAUI/Cartesian-Charts/Axis/Axislabels.md:173]: @Sowndharya1911 update enum values with hyperlink. Will include API links once it is published

File: MAUI/Cartesian-Charts/Axis/Axislabels.md [hunk 2]
```diff
@@ -166,4 +166,79 @@ chart.XAxes.Add(primaryAxis);
...
+{% tabs %}
+
+{% highlight xaml %}
+
+<chart:SfCartesianChart>
+. . .
+    <chart:SfCartesianChart.XAxes>
+      <chart:CategoryAxis LabelsIntersectAction="Wrap">
+         <chart:CategoryAxis.LabelStyle >
+             <chart:ChartAxisLabelStyle  MaxWidth="20" WrappedLabelAlignment="Center" />
```
Comment (VimalaThirumalaiKumarSF3739) [inline MAUI/Cartesian-Charts/Axis/Axislabels.md:217]: @Sowndharya1911 as MaxWidth & WrappedLabelAlignment API exposed in axis LabelStyle class, update it under LabelStyle content. For wrap option, we can mention as notes

Comment (Sowndharya1911) [reply MAUI/Cartesian-Charts/Axis/Axislabels.md]: @VimalaThirumalaiKumarSF3739  Addressed the suggestions

**Review by Saravanan-Madhesh — APPROVED** [review]
Fine

**Pull request merged**